drivers/storage/portworx/component/security.go

```diff
@@ -138,10 +185,9 @@ func (c *security) createPrivateKeysSecret(
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      pxutil.SecurityPXAuthKeysSecretName,
 			Namespace: cluster.Namespace,
-		},
-		StringData: map[string]string{
-			SecuritySharedSecretKey: sharedSecretKey,
-			SecuritySystemSecretKey: internalSystemSecretKey,
+		}, Data: map[string][]byte{
+			SecuritySharedSecretKey: []byte(sharedSecretKey),
+			SecuritySystemSecretKey: []byte(internalSystemSecretKey),
```

Comment:
We will be separating these into two secrets later, right?

Reply:
Yes I'll do this in a separate PR